### DISCUSSION TITLE
Rating Undo?

### DISCUSSION THREAD

**Discussion by RowdyAtl** [TumblThreeApp/TumblThree#502] [Q&A (Question & Answer)]
How do I remove 1-star ratings?

**Comment by thomas694** [comment]
It seems that it's not possible to remove a rating at the moment except editing the index file.

**Comment by thomas694** [comment]
It will be fixed in the next version.

**Comment by RowdyAtl** [comment]
Awesome.  Thank you and for the great app. Sent from my iPhoneOn Apr 24, 2024, at 4:48 PM, Thomas ***@***.***> wrote:﻿
It will be fixed in the next version.
